### ISSUE TITLE
Reddit API wrapper

### ISSUE THREAD

**Issue by fisher60** [fisher60/Friendo_Bot#46]
Create api wrapper for reddit to pull some memes from related topics. This should serve a single meme upon request (with a command)

**Comment by bislara** [comment]
Hey, I would like to work on this for Hacktoberfest 2020. Can you please assign it to me?

**Comment by fisher60** [comment]
Awesome! Great to here, I have assigned you. Let me know if you have any questions.

This bot is for a programming server. So if you need topics, anything related to tech, programming, or anything funny is fine for topics.

**Comment by Ajay-Singh-Rana** [comment]
Is this still open..? I'd like to work on it.

**Comment by fisher60** [comment]
@Ajay-Singh-Rana yes, this issue is open, feel free to claim it, I can assign you if you would like. Please let me know if you have any questions.
